phoenix-pherf/src/main/java/org/apache/phoenix/pherf/workload/mt/operations/TenantOperationFactory.java

```diff
@@ -164,7 +175,7 @@ public void funnel(TenantView tenantView, PrimitiveSink into) {
         }
 
         // This holds the info whether the tenant view was created (initialized) or not.
-        return BloomFilter.create(tenantViewFunnel, numTenants, 0.01);
+        return BloomFilter.create(tenantViewFunnel, numTenants, 0.0000001);
```

Comment:
why did we change from `0.01` to `0.0000001`?

Reply:
https://hur.st/bloomfilter/?n=1000000&p=1.0E-7&m=&k=
This shows that the space  requirements are not that large for 1M items, for that false-positive probability